Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN chmod +x /usr/bin/aws-iam-authenticator
 
 RUN wget https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm 
 RUN chmod +x /usr/local/bin/helm
+RUN helm plugin install https://github.com/zendesk/helm-secrets
```

Comment:
Please remove this, as the action has been factored to be plugins-agnostic and let the user what needs to be installed